src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const DAILY_RENTAL_RATE = 40;
+  const LONG_TERM_DAYS = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const MEDIUM_TERM_DAYS = 3;
+  const MEDIUM_TERM_DISCOUNT = 20;
+
+  const baseCost = days * DAILY_RENTAL_RATE;
+
+  if (days >= LONG_TERM_DAYS) {
+    return baseCost - LONG_TERM_DISCOUNT;
+  }
+
+  if (days >= MEDIUM_TERM_DAYS) {
+    return baseCost - MEDIUM_TERM_DISCOUNT;
```

Comment:
The current logic applies the medium-term discount even when the long-term discount condition is met. To fix this, consider reordering the conditions or adjusting the logic so that only the largest applicable discount is applied. For example, you could calculate the discount separately and then apply it after checking all conditions.